api/ceramic.yaml

```diff
@@ -79,8 +90,18 @@ components:
       description: Event to add to the node
       required: true
   schemas:
+    Version:
+      title: Version
+      description: Version of the ceramic server
+      type: object
+      required:
+        - version
+      properties:
+        version:
+          type: string
+          description: Version of the ceramic server
```

Comment:
Are we making any guarantees as to the content of this version string? Is it semver? Something else. Can they be compared in any way?

Reply:
It is semver. Do we want to define a semver type, or just say the string is semver in description?

Reply:
Semver in the description is good enough for me.